src/vivarium_public_health/risks/effect.py

```diff
@@ -40,34 +43,37 @@ class RiskEffect(Component):
 
     """
 
-    CONFIGURATION_DEFAULTS = {
-        "effect_of_risk_on_target": {
-            "measure": {
-                "relative_risk": None,
-                "mean": None,
-                "se": None,
-                "log_mean": None,
-                "log_se": None,
-                "tau_squared": None,
-            }
-        }
-    }
-
-    ##############
+    ###############
     # Properties #
     ##############
 
+    @property
+    def name(self) -> str:
+        return self.get_name(self.risk, self.target)
+
+    @staticmethod
+    def get_name(risk: EntityString, target: TargetString) -> str:
+        return f"risk_effect.{risk.name}_on_{target.name}"
+
     @property
     def configuration_defaults(self) -> Dict[str, Any]:
         """
         A dictionary containing the defaults for any configurations managed by
         this component.
         """
         return {
-            f"effect_of_{self.risk.name}_on_{self.target.name}": {
-                self.target.measure: self.CONFIGURATION_DEFAULTS["effect_of_risk_on_target"][
-                    "measure"
-                ]
+            self.name: {
+                "data_sources": {
+                    "relative_risk": "self::get_relative_risk_source",
```

Comment:
@rmudambi The way we have to call these methods in build lookup table makes me think this is either not configurable or I should have parsed them from the configuration when we call them in that method

Reply:
I'm not sure I understand this comment. This is configurable, as you could write a different function to get the data, or you could just provide an artifact key to load the data directly, if that data was already in the correct format.